app/views/shared/_header_navbar.html.erb

```diff
@@ -22,5 +22,13 @@
         <div class="h1 my-3"><a href="/">Course Reserves</a></div>
       </div>
     </div>
+    <div class="navbar collapse navbar-collapse"
```

Comment:
Why does this need an `aria-label`?

Reply:
Because it is in the component library. Should we be removing that?

Reply:
In the component library, the labeled element is a `<nav>` which requires an aria-label. https://github.com/sul-dlss/component-library/blob/a16a38a2ae967f5ca72aa7226271e708822e675b/header/dark_variant.html#L216